web/app/scripts/stepwise/stepwise-directive.js

```diff
@@ -52,52 +62,64 @@
                         .append('g')
                         .attr('class', 'outer')
                         .attr('transform', 'translate(' + margin.left + ',' + margin.top +')');
-                }
 
-                /**
-                 * Update all fields of chart with new information and draw in cells with events
-                 *
-                 * @param data {array} the data to change our chart with
-                 */
-                function updateChart(data) {
-                    y.domain([0, d3.max(data, function(d) { return d.count; })]);
+                    svg.append('path')
+                        .attr('class', 'line')
+                        .attr('stroke', 'black');
+
                     xAxis = d3.svg.axis()
                         .scale(x)
-                        .tickFormat(xAxisTextFormat)
                         .orient('bottom')
                         .tickSize(1);
 
                     yAxis = d3.svg.axis()
                         .scale(y)
                         .orient('left')
-                        .ticks(Math.min(10, _.max(_.map(data, function(d) { return d.count; }))))
                         .tickSize(1);
 
                     svg.append('g')
-                      .attr('class', 'x axis')
-                      .call(xAxis)
+                      .attr('class', 'xAxis')
+                      //.call(xAxis)  // TODO: uncomment after worldbank demo
```

Comment:
What happens when we eventually uncomment this? Does it break anything in particular?